microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -162,6 +162,14 @@ Previously, the {microshift-short} load balancer controller tried to update the
 
 Previously, when `microshift-etcd` unexpectedly exited, {microshift-short} tried to restart so that `microshift-etcd` could restart, but there was a lingering unit fragment. Every attempt to restart `microshift-etcd` failed, making the system unusable. The `--collect` flag was added to the `systemd-run` invocation used to start `microshift-etcd`. The additional flag results in systemd cleaning up the unit fragment even if the unit failed. The system now recovers and restarts. (link:https://issues.redhat.com/browse/OCPBUGS-33588[OCPBUGS-33588])
 
+//check status for next release+
+[id="microshift-4-16-known-issues_{context}"]
+== Known issues
+
+[id="microshift-4-16-pods-writing-files-excess-memory-limits-crash"]
+=== Pods crash when writing files that exceed memory limits
+Because of an issue with {op-system-base}, when a pod tries to write files that are larger than configured memory limits to a persistent volume claim, the pod might crash with an out-of-memory error. The pod status shows `OOMKilled` when this occurs. To avoid this known issue, use the workarounds in the knowledge center, link:https://access.redhat.com/solutions/7076169[Pods writing files larger than memory limit to PVCs tend to OOM frequently running on MicroShift].
```

Comment:
[SSG](https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links)  It is called Knowledge Base. So I would keep it simple like below.

```suggestion
Because of an issue with {op-system-base}, when a pod tries to write files that are larger than configured memory limits to a persistent volume claim, the pod might crash with an out-of-memory error. The pod status shows `OOMKilled` when this occurs. To avoid this known issue, see link:https://access.redhat.com/solutions/7076169[Pods writing files larger than memory limit to PVCs tend to OOM frequently running on MicroShift] (Red Hat Knowledgebase).
```